services/ui_backend_service/data/cache/generate_dag_action.py

```diff
@@ -133,7 +133,8 @@ def generate_dag(flow_id, tarball_path):
             'type': node.type,
             'box_next': node.type not in ('linear', 'join'),
             'box_ends': node.matching_join,
-            'next': node.out_funcs
+            'next': node.out_funcs,
+            'doc': node.doc
```

Comment:
Is it possible here that node.doc is not defined or is None?

Reply:
Meaning should we have fallback for that?

Reply:
It will always default to empty string, this is handled here: https://github.com/Netflix/metaflow-service/blob/44e724600e5fdee0b76850eafe31d70fe79f993f/services/ui_backend_service/data/cache/custom_flowgraph.py#L158